src/utils/transaction.ts

```diff
@@ -2160,15 +2155,14 @@ export const getCrossChainStatusByHash = async (
   fromChainId: number,
   toChainId: number,
   hash: string,
-  bridge?: string
 ): Promise<LiFiStatus | null> => {
   if (!sdk) return null;
   try {
     const options = { method: 'GET', headers: { accept: 'application/json' } };
 
     const result = await (
       await fetch(
-        `https://li.quest/v1/status?bridge=${bridge}&fromChain=${fromChainId}&toChain=${toChainId}&txHash=${hash}`,
+        `https://li.quest/v1/status?fromChain=${fromChainId}&toChain=${toChainId}&txHash=${hash}`,
```

Comment:
works with any bridge and we're still providing tx hash as unique identifier so less error prone and does not require additional excess code for `li.quest` supported `bridge` IDs